src/pages/explore/dfi/_components/HarnessDFIGrid.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  VarietyIcon,
+  VarietyIconHover,
+} from "@components/icons/assets/exploreGetDFI/VarietyIcon";
+import {
+  CollateralIcon,
+  CollateralIconHover,
+} from "@components/icons/assets/exploreGetDFI/CollateralIcon";
+import {
+  NodesIcon,
+  NodesIconHover,
+} from "@components/icons/assets/exploreGetDFI/NodesIcon";
+import {
+  TokenizationIcon,
+  TokenizationIconHover,
+} from "@components/icons/assets/exploreGetDFI/TokenizationIcon";
+import {
+  CommunityIcon,
+  CommunityIconHover,
+} from "@components/icons/assets/exploreGetDFI/CommunityIcon";
+import {
+  LPIcon,
+  LPIconHover,
+} from "@components/icons/assets/exploreGetDFI/LPIcon";
+import classNames from "classnames";
+
+interface HarnessDFIGridProps {
+  title: string;
+  desc: string;
+  icon: HarnessDFIICONS;
+  id?: string;
+}
+
+export function HarnessDFIGrid({
+  gridItems,
+  isMobile = false,
+}: {
+  gridItems: HarnessDFIGridProps[];
+  isMobile?: boolean;
+}) {
+  return (
+    <div
+      className={classNames(
+        "grid grid-flow-row ",
+        "lg:grid-rows-2 lg:grid-cols-3 lg:gap-[89.5px]",
+        "md:grid-rows-3 md:grid-cols-2 md:gap-x-[89.5px] md:gap-y-[75px]",
+        "gap-y-12"
+      )}
+    >
+      {gridItems.map((item, index) => (
+        <HarnessDFIGridItem
+          key={index}
+          title={item.title}
+          desc={item.desc}
+          icon={item.icon}
+          id={isMobile ? `${item.icon}-mobile` : `${item.icon}`}
+        />
+      ))}
+    </div>
+  );
+}
+
+function HarnessDFIGridItem({ title, desc, icon, id }: HarnessDFIGridProps) {
+  const Icon = iconMapping[icon];
+
+  // Todo: to uncomment when animations are ready
+  // const HoverIcon = iconMapping[`${icon + 1}`];
+  // const [isHovering, setIsHovering] = useState(false);
+
+  return (
+    <div
+      // onMouseEnter={() => setIsHovering(true)}
+      // onMouseLeave={() => setIsHovering(false)}
+      className={classNames("flex flex-col md:w-[246px] w-full gap-y-4")}
```

Comment:
the space between icon and the label is 20px

Reply:
updated